components/menu/src/Menu.svelte

```diff
@@ -0,0 +1,148 @@
+<script lang="ts">
+  import { slide } from "svelte/transition";
+  import { getNodeAttribute } from "@wetix/utils";
+  import type { MenuItems } from "../types";
+
+  let className = "";
+  export { className as class };
+
+  export let items: MenuItems[] = [];
+  export let level = [];
+  export let onSelectMenu = (_) => {};
+
+  const handleSelectMenu = (e: Event) => {
+    const data = JSON.parse(getNodeAttribute(e, "data-item"));
+    onSelectMenu(data);
+  };
+
+  const isActive = (href: string): boolean =>
+    `${window.location.pathname}${window.location.hash}${window.location.search}` === href;
```

Comment:
we shouldn't handle the location path, should leave it to user to handle using `on:change` event

Reply:
Replaced with `isActive` attribute in items.